callisto/delivery/forms.py

```diff
@@ -62,11 +74,7 @@ class SecretKeyForm(forms.Form):
         'wrong_key': "The passphrase didn't match.",
     }
 
-    key = forms.CharField(max_length=254,
-                          label="Your passphrase",
-                          widget=forms.PasswordInput(attrs={'placeholder': 'Your passphrase',
-                                                            'autocomplete': 'off'}),
-                          error_messages={'required': REQUIRED_ERROR.format("passphrase")})
+    key = make_key(254)
```

Comment:
WHOA I did not notice these were different! That's a mistake, since they all accept the same data. Can you take out the length parameter and just set it to 64 inside `make_key`?